alembic/versions/f355b5458f82_bug_1474037_firefoxreality.py

```diff
@@ -0,0 +1,34 @@
+"""bug 1474037: add FirefoxReality
+
+Revision ID: f355b5458f82
+Revises: fab476fc2f5e
+Create Date: 2018-08-29 13:51:49.319243
+
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'f355b5458f82'
+down_revision = 'fab476fc2f5e'
+
+
+PRODUCTNAME = 'FirefoxReality'
+SORT = 4
+
+
+def upgrade():
+    op.execute("""
+        INSERT INTO products
+        (product_name, sort, release_name, rapid_beta_version, rapid_release_version)
+        VALUES
+        ('%s', %s, '%s', 1.0, 1.0)
+    """ % (PRODUCTNAME, SORT, PRODUCTNAME.lower()))
```

Comment:
Alembic's `op.execute` doesn't support parameters. We could use their insert function which generates the insert statement. I looked at that for a bit and decided it's less risky for us to use something that's not deeply abstracted even if it involves interpolation of variables into SQL.